featuretools/io/sql.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+
+
+def from_sql(id, connection=None, connection_string=None, tables=None, index={},
+             time_index={}, secondary_time_index={}, encoding={}, variable_types={}):
+    if connection is None:
+        if connection_string is None:
+            raise ValueError('connection or connection_string is required')
+        else:
+            connection = sa.create_engine(connection_string, echo=False).connect()
+
+    loader = EntitySetLoaderFromSQL(connection=connection, id=id, index=index, time_index=time_index,
+                                    secondary_time_index=secondary_time_index, encoding=encoding,
+                                    variable_types=variable_types)
+    loader.load_entityset(tables)
+    return loader.es
+
+
+def _find_pk_name(table):
+    column_names = table.columns.keys()
+    pks = [name for name in column_names if table.columns[name].primary_key]
+    if len(pks) > 1:
+        raise RuntimeError('Composite primary key detected')
+    elif len(pks) == 1:
+        return pks[0]
+    else:
+        return None
+
+
+class EntitySetLoaderFromSQL:
+    def __init__(self, id, connection, **passthrough_args):
+        self.connection = connection
+        self.metadata = sa.MetaData(connection)
+        self.metadata.reflect(bind=self.connection.engine)
+        self.passthrough_args = passthrough_args
+        self.table_names = self._get_table_names()
+        self.es = ft.EntitySet(id=id)
+
+    def load_entityset(self, tables):
+        self.load_tables(tables)
+        self.load_relationships()
+
+    def _get_table_names(self):
+        return list(self.metadata.tables.keys())
+
+    def _get_args_for_table(self, table_name):
+        return {k: v.get(table_name) for k, v in self.passthrough_args.items()}
+
+    def _load_table(self, table_name):
+        table = sa.Table(table_name, self.metadata, autoload=True)
+        df = pd.read_sql(table.select(), table.metadata.bind)
+
+        table_args = self._get_args_for_table(table_name)
+        print("table_args:", table_args)
```

Comment:
either remove this print statement or wrap it in `if verbose:` like we do in other places